src/equisolve/numpy/preprocessing/_base.py

```diff
@@ -121,9 +130,7 @@ def fit(
                 if self.with_mean:
                     mean_values = np.average(X_mat, weights=sample_weights, axis=0)
                 else:
-                    mean_values = np.zeros(
-                        self.n_properties_.block(key).values.flatten()
-                    )
+                    mean_values = np.zeros(self.n_properties_)
```

Comment:
unrelated to this PR, but why is the StandardScaler not implemented in terms of equistore operations? It looks like it is re-doing what should be `equistore.mean`, `equistore.std`, `equistore.substract`, `equistore.div`. Is something missing in the operations?

Reply:
because these were not implemented when we implemented the class

Reply:
fair enough 😄 I'll open an issue to update this =)

Reply:
made an issue #18